### PR TITLE
[dev] Responsive navbar

### PR DESCRIPTION
**Pull request by Joanartdacosta**
navbar for desktop
navbar for mobile
css code to hidden and show the component according viewporth
add responsive design to images of about me page

### CODE REVIEW

File: app/src/components/common/navbar/NavBar.js [hunk 1]
```diff
@@ -1,32 +1,17 @@
...
-        >
-          SERVIÇOS
-        </Link>
-        <Link
-          className="text-xl text-black no-underline pr-10"
-          href="/projects"
-        >
-          PROJETOS
-        </Link>
+    <div>
```
Comment (doublethink13) [inline app/src/components/common/navbar/NavBar.js:6]: is this div necessary?

File: app/src/components/common/navbar/NavBar.js [hunk 2]
```diff
@@ -1,32 +1,17 @@
...
+      <div>
+        <div className="menu-desktop">
```
Comment (doublethink13) [inline app/src/components/common/navbar/NavBar.js:8]: im not sure that actually having both navbars here is the best practice

where did you get the inspiration for this?

this seems a better approach: https://blog.logrocket.com/create-responsive-navbar-react-css/

you can also use some sort of variable to conditionally render the navbar you need

File: app/src/components/common/navbar/NavBar.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/src/components/common/navbar/NavBar.js:6]: yes, to apply flexbox effect

File: app/src/components/common/navbar/NavBar.js (hunk 2 above)
Comment (Joanartdacosta) [reply app/src/components/common/navbar/NavBar.js]: here: https://khuang159.medium.com/creating-a-hamburger-menu-in-react-f22e5ae442cb

The approach referred in your suggestion demands the installation of a new package react-responsive and I decided to proceed with a simpler option (no package needed). However, I am going to investigate more options

File: app/src/components/common/navbar/NavBar.js [hunk 3]
```diff
@@ -1,32 +1,26 @@
-import Link from "next/link";
+import NavLink from "../link/NavLink";
```
Comment (doublethink13) [inline app/src/components/common/navbar/NavBar.js:1]: @

Comment (Joanartdacosta) [reply app/src/components/common/navbar/NavBar.js]: well noted, thank you!
All the imports were reviewed and corrected

**Review by doublethink13 — APPROVED** [review]

**Pull request merged**